demos/human_pose_estimation_demo/src/human_pose_estimator.cpp

```diff
@@ -34,10 +34,7 @@ HumanPoseEstimator::HumanPoseEstimator(const std::string& modelPath,
         ie.SetConfig({{InferenceEngine::PluginConfigParams::KEY_PERF_COUNT,
                        InferenceEngine::PluginConfigParams::YES}});
     }
-    netReader.ReadNetwork(modelPath);
-    std::string binFileName = fileNameNoExt(modelPath) + ".bin";
-    netReader.ReadWeights(binFileName);
-    network = netReader.getNetwork();
+    auto network = ie.ReadNetwork(modelPath);
```

Comment:
```suggestion
    network = ie.ReadNetwork(modelPath);
```

You've shadowed the member variable with the same name.